class.jetpack-xmlrpc-server.php

```diff
@@ -177,52 +184,121 @@ public function remote_provision( $request ) {
 					__( 'There was an issue validating this request.', 'jetpack' ),
 					400
 				),
-				'jpc_remote_provision_fail'
+				'jpc_remote_register_fail'
 			);
 		}
 
-		$local_username = $request['local_username'];
+		if ( ! Jetpack_Options::get_option( 'id' ) || ! Jetpack_Options::get_option( 'blog_token' ) || ! empty( $request['force'] ) ) {
```

Comment:
We should probably check if this is truthy, not whether it's empty.

Reply:
I think that `empty()` will work well in this case since it'll check that the key is there as well as check if the value is falsey. Here's a note from the PHP docs.

> empty() is essentially the concise equivalent to !isset($var) || $var == false.

Reply:
TIL!